From c95149b486ba8bbc69ee5027aaaf2417d0d8c048 Mon Sep 17 00:00:00 2001
From: Tulip Blossom <tulilirockz@posteo.com>
Date: Sat, 11 Oct 2025 11:37:37 -0300
Subject: [PATCH] nosystemd

This just removes the only place that actually requires systemd in the entire thing

Signed-off-by: Tulip Blossom <tulilirockz@posteo.com>
---
 meson.build | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/meson.build b/meson.build
index b736b9c..5db435c 100644
--- a/meson.build
+++ b/meson.build
@@ -25,11 +25,6 @@ libdrm_nouveau = dependency('libdrm_nouveau', version: '>= 2.4.97', required: ge
 libdrm_amdgpu = dependency('libdrm_amdgpu', version: '>= 2.4.97', required: get_option('libdrm_amdgpu'))
 drm_xe = cc.has_header('drm/xe_drm.h', include_directories: include_directories(libdrm.get_pkgconfig_variable('includedir')), dependencies: [libdrm], required: get_option('drm_xe'))
 
-systemd_systemunitdir = get_option('systemdsystemunitdir')
-if systemd_systemunitdir == ''
-  systemd_systemunitdir = dependency('systemd').get_pkgconfig_variable('systemdsystemunitdir')
-endif
-
 hwdb_dir = get_option('hwdbdir')
 if hwdb_dir == ''
   udevdir = dependency('udev').get_pkgconfig_variable('udevdir')
-- 
2.51.0

